docs/ref/models/expressions.txt

```diff
@@ -450,6 +450,116 @@ Conditional expressions allow you to use :keyword:`if` ... :keyword:`elif` ...
 :keyword:`else` logic in queries. Django natively supports SQL ``CASE``
 expressions. For more details see :doc:`conditional-expressions`.
 
+``Subquery()`` expressions
+--------------------------
+
+.. currentmodule:: django.db.models.expressions
+
+.. class:: Subquery(queryset, output_field=None)
```

Comment:
I think it should be queryset, because that shows what type of argument is expected.